pocs/tests/test_pocs.py

```diff
@@ -324,6 +324,38 @@ def test_unsafe_park(pocs):
     pocs.power_down()
 
 
+def test_no_ac_power(pocs):
+    # Simulator makes AC power safe
+    assert pocs.has_ac_power() is True
+
+    # Remove 'power' from simulator
+    pocs.config['simulator'] = ['camera', 'mount', 'weather']
+    pocs.initialize()
+
+    # With simulator removed the power should fail
```

Comment:
Could also insert into the test db a current record for power showing a failure (or success). That way more of the code is evaluated.

Reply:
I enter a range of values in the loop that follows this. Were you thinking of something different?